audit/audit.cpp

```diff
@@ -113,7 +131,23 @@ audit::report_error(const std::string& short_name, const std::string& descriptio
 {
     this->recorded_errors.push_back(description);
     LOG(fatal) << boost::format("[%1%]: %2%") % short_name % description;
-    // TODO: Push to stats.d goes here
+    this->send_to_monitor(str(boost::format("%1%:1|c") % short_name));
```

Comment:
It may be better to put the format in a "const std::string". 
In the future we may have more types such as guages, timing etc.